### PR TITLE
Sync ci-operator image to build01

### PR DESCRIPTION
**Pull request by hongkailiu**
follow up https://github.com/openshift/ci-tools/pull/261

/cc @openshift/openshift-team-developer-productivity-test-platform

### CODE REVIEW

**Comment by hongkailiu** [comment]
@droslean 
Any idea?
https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5825/pull-ci-openshift-release-master-generated-config/11543

**Comment by petr-muller** [comment]
@hongkailiu that's caused by https://github.com/openshift/ci-tools/pull/262 merging without it;s followup https://github.com/openshift/release/pull/5822. Should be fine now, just retest.

File: clusters/build-clusters/01_cluster/openshift-image-registry/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# openshift-image-registry
+
+## Configuration
+
+Doc: [expose the registry using DefaultRoute](https://docs.openshift.com/container-platform/4.2/registry/securing-exposing-registry.html#registry-exposing-secure-registry-manually_securing-exposing-registry)
```
Comment (petr-muller) [inline clusters/build-clusters/01_cluster/openshift-image-registry/README.md]: Hmm what do we need the expose for? At least for now we'll only need in-cluster access to the images, no?

Comment (hongkailiu) [reply clusters/build-clusters/01_cluster/openshift-image-registry/README.md]: That is correct. We do not have to.
It is convenient to expose it when testing the image. I can pull it with docker/podman.
Do not think it going to harm us badly. On 3.x ... it is exposed by default.

**Pull request merged**